src/plays/popup-component/components/popupCard.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import './styles.css';
+
+function popupCard({ children, popTitle, btnStyle }) {
+  const [toggle, setToggle] = useState(false);
+  const toggleModal = () => {
+    setToggle(!toggle);
+  };
+
+  return (
+    <>
+      <button className="modal__btn" style={btnStyle} onClick={toggleModal}>
+        {popTitle}
+      </button>
+      {toggle && (
+        <div>
+          <div className="overlay" onClick={toggleModal} />
```

Comment:
Use more meaningful class names instead of generic names like `overlay` and `modal_content`